### PR TITLE
Survey

### PR DESCRIPTION
**Pull request by BhagyaC**
Please add your thoughts on pandas contributors survey

### CODE REVIEW

**Comment by datapythonista** [comment]
Thanks for starting this @BhagyaC, looks great.

@python-sprints/pandas-mentoring can everybody interested please have a look at this PR about the survey?

Also, CC'ing Gina, @Dr-G, who is the leader of diversity and inclusion at NumFOCUS (the non-profit that supports pandas), who may be able to help.

**Comment by dujm** [comment]
Hi @ BhagyaC!  I like what you wrote in the survey. Just wonder how we are going to do the survey later? What would you suggest?

**Review by dujm — APPROVED** [review]

**Comment by BhagyaC** [comment]
> Hi @ BhagyaC! I like what you wrote in the survey. Just wonder how we are going to do the survey later? What would you suggest?

Hi @dujm ,
I think first we need to know what are the thought of others on this (is in't it? @datapythonista ).Thats why I created this PR. Once we got that we can move to the next step.To take a survey or gather information will not be a great challenge , a google form will be sufficient(I suppose so , If you have anything in mind please share). But before going to that we have to make sure about the participation and interest and that will be the result of this PR

**Comment by dujm** [comment]
@ BhagyaC, Yes, you are right. This is also how I understood it -- first collect questions. Then I would also go for Google form. 

Just to be clear, I would love to do the survey.

File: SURVEY.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+# Pandas Devolopers Survey
+
+**Hello folks ,
+
+We need a survey to gather information about the pandas dev. The survey must include information about  general pandas usage, types of development, technologies, and demographics etc.
```
Comment (Dr-G) [inline SURVEY.md]: Can you clarify who the survey will be distributed to? Is it _all_ pandas contributors?

File: SURVEY.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+Reliability and stability
+Usability 
+
+### Which of these future capabilities of pandas would you be most excited about?
+
+Method chaining 
+Apache Arrow
+Extension Arrays
+
+### Employment status
```
Comment (Dr-G) [inline SURVEY.md:50]: Why do we want to know this?

File: SURVEY.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+Data analyst
+Technical support
+Systems analystCIO / CEO / CTO
+QA engineer
+Product manager
+Business analyst
+DBA
+Other
+
+### Working in a Team vs Working Independently
```
Comment (Dr-G) [inline SURVEY.md:73]: Why do we want to know this?

File: SURVEY.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+### Team Size
+
+2–7 people
+8–12 people
+13–20 people
+21–40 people
+More than 40 people
+
+### Company Size
```
Comment (Dr-G) [inline SURVEY.md:87]: Why do we want to know this?

**Comment by Dr-G** [comment]
Hi folks,

Thanks for including me! We've done a good deal of survey development for NumFOCUS in the past; I'm happy to help shape this for pandas.

Can someone please clarify:
- who you are planning to send the survey to
- what you are hoping to learn from the survey
- how the findings from the survey will be acted upon

Thanks!

**Comment by datapythonista** [comment]
Since this was merged before having much discussion, may be we should create an issue to track the proposed updates to this draft?

@Dr-G the answers to your questions are still open and under discussion. But the original idea and motivation was to survey the people in this mentoring program (around 25 people). To learn about each other, and to have a bit more explicit information about the background and previous experience of everybody. We're discovering that information as we work together, but if for example just couple of people don't have previous experience with git, they can be feeling this is not for them, and would be nice that everybody is aware, and we can better help them catch up.

Finally, would be interesting to gather information on whether people could be interested in scholarships. I assume many people will have full time jobs and prefer to contribute as a volunteer in their free time, while others can be students who could enjoy a scholarship for  the summer time, and I guess other people are self-employeed, have part-time jobs, or are not working, and could be interested in contributing as a paid job. The information can be relevant to work and decide on funding options (corporate sponsors, NumFOCUS small grants, GSoC, pandas funds...). Also we're preparing the application for a pandas CZI grant, and it may be relevant to add to the proposal if we have a specific program to increase the diversity among pandas developers that the grant could fund.

That's the original idea, from there, everybody is welcome to discuss about the survey, and if people is happy with that, the information could be used by NumFOCUS diversity programs, if you think it can be useful.

**Comment by BhagyaC** [comment]
Hi @Dr-G , 
I hope all your questions are answered by @datapythonista .
And the questions are not finalised . I'll create an issue to the track. 
Thanks

**Comment by Dr-G** [comment]
Thanks, @datapythonista; that sounds good and the context is helpful.

**Pull request merged**